### PR TITLE
docs: add how to videos for playwright

### PR DESCRIPTION
**Pull request by kiran-redhat**
# Description

### <samp>🤖 Generated by Copilot at 508fc02</samp>

This change adds a new document to the docs app that explains how to use `Playwright` for end-to-end testing. The document includes videos and a command to run the tests for the `journeys-e2e` project.

- Link to Basecamp Todo
- https://3.basecamp.com/3105655/buckets/33850877/todos/6552204705

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Check that videos in the docs are working https://docs-1887-jesusfilm.vercel.app/docs/basics/e2e-testing/core-concepts
# Walkthrough

### <samp>🤖 Generated by Copilot at 508fc02</samp>

*  Add a new document about Playwright testing to the docs app ([link](https://github.com/JesusFilm/core/pull/1887/files?diff=unified&w=0#diff-74d46fabb5c4f8a4e2148b2ca9eb4e426e6074184d7f999b460a3ea1324e9c68L1-R82))

### CODE REVIEW

File: apps/docs/docs/02-getting-started/05-playwright-testing.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
+---
+title: 5. Playwright Testing
```
Comment (storyworks) [inline apps/docs/docs/02-getting-started/05-playwright-testing.mdx:2]: image

Rename this to E2E Testing and move this section to "Basics" as a sibling to "Back-end" and "Front-end"?

**Comment by tataihono** [comment]
@kiran-redhat thanks for making the videos! If you could also do a brief write-up under each video for our brothers and sisters who might be on slower network connections when developing.

File: apps/docs/docs/02-getting-started/05-playwright-testing.mdx (hunk 1 above)
Comment (kiran-redhat) [reply apps/docs/docs/02-getting-started/05-playwright-testing.mdx]: This looks like a perfect place. Moved here.

**Comment by kiran-redhat** [comment]
> @kiran-redhat thanks for making the videos! If you could also do a brief write-up under each video for our brothers and sisters who might be on slower network connections when developing.

@tataihono , I've now split one page into three different sections. How does that look?

**Review by mikeallisonJS — APPROVED** [review]

**Pull request merged**